### PR TITLE
Add Easy!Appointments

### PR DESCRIPTION
**Pull request by ax2bboud**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Your submissions are formatted according to the following requirements:
        
``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Depends on proprietary services:

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Non-free software in `non-free.md`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [ ] Your additions are ordered alphabetically.
- [ ] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [ ] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.
- [ ] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [ ] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [ ] Any licenses you have added are in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [ ] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [ ] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
- [ ] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [ ] Any software project you are adding to the list is actively maintained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,6 +253,7 @@ _CalDAV or CardDAV servers_
 - [DAViCal](https://www.davical.org/) - Server for calendar sharing (CalDAV) that uses a PostgreSQL database as a data store. ([Source Code](https://gitlab.com/davical-project/davical)) `GPL-2.0` `PHP`
 - [Davis](https://github.com/tchapi/davis/) - A simple, dockerizable and fully translatable admin interface for sabre/dav based on Symfony 5 and Bootstrap 4, largely inspired by Baïkal. `MIT` `PHP`
 - [DecSync CC](https://f-droid.org/packages/org.decsync.cc/) - Serverless contacts, calendar synchronization using your own file syncing method i.e Syncthing, Nextcloud etc. ([Source Code](https://github.com/39aldo39/DecSyncCC)) `GPL-3.0` `Kotlin`
+- [Easy!Appointments](https://easyappointments.org/) - Easy!Appointments is a highly customizable web application that allows your customers to book appointments with you via the web. Supports Google Calendar syncing. ([Demo](https://demo.easyappointments.org/), [Source Code](https://github.com/alextselegidis/easyappointments)) `GPL-3.0` `PHP`
```
Comment (SuperSandro2000) [inline README.md]: the Demo an main link are duplicated.

Comment (ax2bboud) [reply README.md]: Hey @SuperSandro2000 . I thought sub-domains would count as a separate URL for this purpose.

Comment (SuperSandro2000) [reply README.md]: Oh, you are right. I didn't catch the demo part. Sorry about that.

**Comment by n8225** [comment]
Closing PR Easy!Appointments is already on the list https://github.com/awesome-selfhosted/awesome-selfhosted#booking-and-scheduling

**Pull request closed without merging**